include/vk_mem_alloc.h

```diff
@@ -16675,7 +16675,7 @@ VMA_CALL_PRE VkResult VMA_CALL_POST vmaGetMemoryWin32Handle(VmaAllocator VMA_NOT
     VMA_DEBUG_GLOBAL_MUTEX_LOCK;
     return allocation->GetWin32Handle(allocator, hTargetProcess, pHandle);
 }
-#endif // VMA_EXTERNAL_MEMORY_WIN32 
+#endif // VMA_EXTERNAL_MEMORY_WIN32
```

Comment:
It seems that you change many lines that are unrelated to your change due to using some automated formatting tools. Changes like this make it difficult to analyze the history of a file later (like "blame" function).